src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.scss

```diff
@@ -81,6 +48,54 @@
   }
 }
 
+.labelTextPrimary {
+  margin-right: 10px;
+}
+
+.main {
+  .labelTextPrimary {
+    font-weight: $bold;
+  }
+}
+
+// TODO: would sure be nice to rename this class
+.selectedInfo {
+  color: $black;
+  font-size: 10px;
+  font-weight: $normal;
+  margin-right: 10px;
+}
+
+// TODO: would sure be nice to rename this class
+.additionalInfo {
```

Comment:
Can we change this to type? This name would be tricky if we don't consider strand to be a type though.